projects/storefrontapp-e2e-cypress/cypress/helpers/cart-import-export.ts

```diff
@@ -256,10 +256,16 @@ export function addProductToCart(productCode: string = cart.products[1].code) {
  * @param expectedData will compare the data of the downloaded .csv to the parsed string.
  */
 export function exportCart(expectedData?: string) {
+  const DOWNLOADS_FOLDER = Cypress.config('downloadsFolder');
+  cy.task('deleteFolder', DOWNLOADS_FOLDER);
+
   const cartPage = waitForPage('/cart', 'getCartPage');
   cy.visit('/cart');
   cy.wait(`@${cartPage}`).its('response.statusCode').should('eq', 200);
   cy.get('cx-export-order-entries button').contains('Export to CSV').click();
```

Comment:
before the test just click "export" button, then check the downloaded file immediately. It is possible that the file downloading is not complete yet. (In this case, the old `cart.csv` file will be read). So, in this function, before download file, I remove the old downloaded files.